### PR TITLE
[add/update][l] Updating the dataset

### PR DESCRIPTION
**Pull request by gradedSystem**
## Changes made:

Updated the CDIAC dataset from old 2014 data to 2020 data (it doesn't seem like getting updated very frequently so better to update it once in 2-3 years when new data would be available)

- Updated `datapackage.json`
- Updated `README.md`
- Updated the `scripts/`, `cache/` and `data/` folders with new values and scripts

cc @anuveyatsu

### CODE REVIEW

File: datapackage.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "co2-fossil-by-nation",
-  "title": "CO2 Emissions from Fossil Fuels since 1751, By Nation",
+  "title": "National CO2 emissions from fossil fuels and cement manufacture",
```
Comment (anuveyatsu) [inline datapackage.json:3]: @gradedSystem I'm not sure if we should update the title. But I wanted to confirm if this new source actually includes co2 emission data from cement manufacture?

File: datapackage.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   "sources": [
     {
-      "name": "CDIAC",
-      "path": "http://cdiac.esd.ornl.gov/ftp/ndp030/CSV-FILES/nation.1751_2011.csv",
-      "title": "CO2 Emissions from Fossil Fuels since 1751, By Nation"
+      "name": "CDIAC at AppState",
+      "path": "https://rieee.appstate.edu/wp-content/uploads/2024/07/nation.1751_2020.xlsx",
+      "title": "National CO2 emissions from fossil fuels and cement manufacture"
```
Comment (anuveyatsu) [inline datapackage.json:8]: @gradedSystem are you copying it from their site?

**Review by anuveyatsu — COMMENTED** [review]
@gradedSystem I would like to understand if the table schema is changing due to the new data source.

File: datapackage.json (hunk 2 above)
Comment (gradedSystem) [reply datapackage.json:8]: @anuveyatsu Correct I am copying the description from their website

File: datapackage.json (hunk 1 above)
Comment (gradedSystem) [reply datapackage.json:3]: @anuveyatsu It includes the data from cement manufacture

**Comment by gradedSystem** [comment]
@anuveyatsu I don't think it changes I made sure it has exactly the same schema

**Pull request merged**